isis/src/base/objs/SpiceRotation/SpiceRotation.h

```diff
@@ -426,7 +424,7 @@ namespace Isis {
       void checkForBinaryPck();
 
       int cacheSize() {
-        return p_cache.size();
+        return m_orientation->getRotations().size();
```

Comment:
This needs to check that the m_orientations pointer is valid. If it's invalid, then it should return 0.

Reply:
Thanks for catching this!